.changeset/dull-crabs-prove.md

```diff
@@ -0,0 +1,5 @@
+---
+"@jspsych-contrib/plugin-image-multi-response": patch
+---
+
+Disable image dragging in plugin-image-multi-response
```

Comment:
I somehow can't edit files in this PR, so here's a suggestion:
```suggestion
Disable image dragging
```
since reading "Disable image dragging in plugin-image-multi-response" in the changelog of `plugin-image-multi-response` feels a bit redundant.